upgrade-instructions.md

```diff
@@ -16,17 +16,36 @@ Changes to apply between 3.4.x and 3.5.x will be reported here.
 
 Change list:
 
+ - [RabbitMQ minimal version](#rabbitmq-minimal-version)
  - [Enforce usernames to be lower cased](#enforce-usernames-to-be-lower-cased)
  - [Cassandra keyspace creation configuration](#cassandra-keyspace-creation-configuration)
  - [UsersFileRepository](#usersfilerepository)
  - [ElasticSearch performance enhancements](#elasticsearch-performance-enhancements)
  - [JAMES-2703 Post 3.4.0 release removals](#james-2703-post-340-release-removals)
 
+#### RabbitMQ minimal version
+
+Date 26/11/2019
+
+SHA-1 0bf4e8384e
+
+Concerned products: Guice distributed James (rabbitMQ)
+
+The distributed James project (relying on Guice, Cassandra, ElasticSearch, RabbitMQ and optionally Swift) benefits from a new distributed task mananger.
+
+In order to enforce task sequential processing at the cluster level, we rely on a single active consumer, which is a feature introduced in RabbitMQ 3.8.
+
+Users of distributed James product thus need to upgrade their RabbitMQ server to be at least version 3.8.
+
+To perform the upgrade the RabbitMQ queue `taskManagerWorkQueue` has to be deleted. James will then recreate it with the single active consumer property.
```

Comment:
`To perform the upgrade the RabbitMQ queue *taskManagerWorkQueue* has to be deleted.`

How?

Reply:
I will remove this fixup, it's useless as it does not concern a released modification.
(for the record, the answer of this question is: using rabbitmq web management interface)